.github/workflows/ci.yml

```
@@ -30,7 +30,7 @@ jobs:
         run: make simulation-test
 
       - name: Go linters
-        uses: golangci/golangci-lint-action@v7
+        uses: golangci/golangci-lint-action@v8
         with:
           version: v2.0.2
```

```suggestion
          version: v2.1.6
```
Per ci, v2.1+ is required.
https://github.com/golangci/golangci-lint says 2.1.6 is current latest.

---

Looks like blocked by https://github.com/openshift/release/pull/64296, can't apply suggestions here via UI or via git cli.